framework/src/interfaces/Coupleable.C

```diff
@@ -1739,6 +1739,23 @@ Coupleable::coupledArrayDofValues(const std::string & var_name, unsigned int com
   return (_c_is_implicit) ? var->dofValuesNeighbor() : var->dofValuesOldNeighbor();
 }
 
+const ADVariableValue &
+Coupleable::adCoupledDofValues(const std::string & var_name, unsigned int comp) const
+{
+  const auto * var = getVarHelper<MooseVariableField<Real>>(var_name, comp);
+
+  if (!var)
+    return *getADDefaultValue(var_name);
+  checkFuncType(var_name, VarType::Ignore, FuncAge::Curr);
+
+  if (!_c_is_implicit)
+    mooseError("Not implemented");
+
+  if (!_coupleable_neighbor)
```

Comment:
I think you’ve got these swapped

Reply:
In the two lines below

Reply:
Hehehe - I originally had `if(_coupleabe_neighbor)` but swapped it to `if(!_coupleabe_neighbor)` to match the existing methods and forgot to swap which one got returned.